utils/compatibility_test.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type rpcChainCompatibility struct {
+	RPCChainVMProtocolVersion map[string]int `json:"rpcChainVMProtocolVersion"`
+}
+
+func TestCompatibility(t *testing.T) {
+	subevmVersion := "v0.4.0"
+	expectedRPCVersion := 17
+
+	compat, err := os.ReadFile("../compatibility.json")
```

Comment:
Can we use JSON embed here? For example: https://github.com/ava-labs/coreth/blob/master/plugin/evm/ext_data_hashes.go#L11-L13

Reply:
`go:embed` only works with files in a subdirectory of the test. I don't think we want to bury this json file in subdirectories, so I'm not going to make this change.